Missionframework/modules/03_persistence/fnc/fn_persistence_saveData.sqf

```diff
@@ -0,0 +1,31 @@
+/*
+    KPLIB_fnc_example_saveData
+
+    File: fn_example_saveData.sqf
+    Author: KP Liberation Dev Team - https://github.com/KillahPotatoes
+    Date: 2019-02-02
+    Last Update: 2019-03-30
+    License: GNU General Public License v3.0 - https://www.gnu.org/licenses/gpl-3.0.html
+
+    Description:
+        Fetches data which is bound to this module and send it to the global save data array.
+
+    Parameter(s):
+        NONE
+
+    Returns:
+        Function reached the end [BOOL]
+*/
+
+if (KPLIB_param_debug) then {diag_log "[KP LIBERATION] [SAVE] Persistence module saving..."};
+
+KPLIB_persistence_objects = KPLIB_persistence_objects - [objNull];
```

Comment:
This line wouldn't be necessary if we use deleteAt instead of setting an index to objNull

Reply:
This is for safety eg. something can be added to persistence but deleted via `deleteVehicle`